public/javascript/cgtpd.js

```diff
@@ -155,7 +155,13 @@
 
   function validateFile(file, form, singleFileUpload, submitForm) {
     if (!file) {
-      return handleFileUploadError(singleFileUpload, "Upload a file containing your supporting evidence")
+      var errorMessage = "";
+      if (singleFileUpload.id === "upload-supporting-evidence")
+        errorMessage = "Upload a document showing how the Capital Gains Tax due was worked out"
```

Comment:
aren't the error messages the wrong way around?

Reply:
this error message should be shown on the upload mandatory evidence page in the YearToDateLiabilityController